### DISCUSSION TITLE
Installation in the right environment

### DISCUSSION THREAD

**Discussion by akvilonBrown** [maartenbreddels/ipyvolume#434] [Q&A] [answered]
Greetings!

I have a question about the installation of ipvolume.
I'm using conda environments with ipkernels and Jupyter is installed in my (base) environment.
Then I'm able to launch specialized environments as kernels in a notebook.
The question is, in which environment should I install ipyvolume: the base or the specialized?

**Comment by maartenbreddels** [comment, marked as answer]
Hi,

both :)
the python code and js bundle are packaged together. Ideally you'd install that separately but it would be an extra maintenance burden for us. If you install in both, it should always work :)

**Reply by akvilonBrown** [reply]
Aha, that makes sense.
That client-side widget and kernel-specific code made me wonder how to marry them.
Thank you!
